source/transfer_list.rst

```diff
@@ -1033,6 +1033,45 @@ passed to later stages for intialisation of Mbed-TLS.
      - hdr_size + 0x8
      - Size of memory region.
 
+**AArch32 executable entry point information (XFERLIST_EXEC_EP_INFO32)**
+
+This entry type holds the 32-bit variant of the `entry_point_info`
+structure.  `entry_point_info` is a TF-A-specific data structure [TF_BL31]_ used
+to represent the execution state of an image; that is, the state of general
+purpose registers, PC, and SPSR.
+
+This information is used by clients to setup the execution environment of
+subsequent images. It's usage is identical to the 64-bit form represented by
+``XFERLIST_EXEC_EP_INFO64``.
+
+.. _tab_entry_point_info32:
+.. list-table:: Entry point info type layout (32-bit variant)
+   :widths: 2 5 2 6
+
+   * - Field
+     - Size (bytes)
+     - Offset (bytes)
+     - Description
+
+   * - tag_id
+     - 0x3
+     - 0x0
+     - The tag_id field must be set to **0x106**.
+
+   * - hdr_size
+     - 0x1
+     - 0x3
+     - |hdr_size_desc|
+
+   * - data_size
+     - 0x4
+     - 0x4
+     - Size of the `entry_point_info` structure in bytes.
+
+   * - ep_info
+     - `sizeof(entry_point_info)`
+     - hdr_size
+     - Holds a single `entry_point_info` structure.
```

Comment:
This structure should be defined here, so please copy over the info from wherever it is kjept.